contracts/test-alt/Marketplace.js

```diff
@@ -643,4 +643,60 @@ describe('Marketplace.sol', async function() {
       }
     })
   })
+
+  describe('Affiliate whitelist', function() {
```

Comment:
could you add a test to ensure that only the owner can add or remove affiliates? I know it's obvious from the modifier, but it's extra piece of mind

Reply:
Actually, I just noticed that you already have tests for this.